src/Core/Merq/CommandFinishedEventArgs.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Merq
+{
+	/// <summary>
+	/// Event args for <see cref="ICommandBus.CommandFinished"/>
+	/// </summary>
+    public class CommandFinishedEventArgs
+    {
+		/// <summary>
+		/// Initialized an instance of <see cref="CommandFinishedEventArgs"/>
+		/// </summary>
+		/// <param name="command">The command</param>
+		/// <param name="error">The exception when the command fails</param>
+		/// <param name="elapsedTime">The amount of time (in ms) the command execution took</param>
+		public CommandFinishedEventArgs(
+			IExecutable command, 
+			Exception error,
+			double elapsedTime)
+		{
+			Command = command;
+			Error = error;
+			ElapsedTime = elapsedTime;
+		}
+
+		/// <summary>
+		/// Gets the command
+		/// </summary>
+		public IExecutable Command { get; }
+
+		/// <summary>
+		/// Gets the exception when the command execution fails
+		/// </summary>
+		public Exception Error { get; }
+
+		/// <summary>
+		/// Gets the amount of time (in ms) the command execution took
+		/// </summary>
+		public double ElapsedTime { get; set; }
```

Comment:
Let's rename to `ElapsedMilliseconds`, which matches what the `Stopwatch` provides and is more evident :)